packages/create-workleap/src/generateProject.ts

```diff
@@ -0,0 +1,55 @@
+import child_process from "node:child_process";
+import process from "node:process";
+import type { TemplateId } from "./templates.js";
+
+interface UserInputs {
```

Comment:
Could we rename this to GenerateProjectArguments?

Reply:
The idea was to also use this type in `bin.ts`.

Something might have been lost in translation here 😆